packages/lib/config/vitest/setupTests.ts

```diff
@@ -0,0 +1,21 @@
+// jest-dom adds custom matchers for asserting on DOM nodes. Works for Vitest too!
+import "@testing-library/jest-dom";
+import "vitest-canvas-mock";
+
+// @ts-ignore
+window.crypto.randomUUID = (() => {
+  let value = 0;
+  return () => {
+    value += 1;
+    return value.toString();
+  };
+})();
```

Comment:
is this block necessary? If not, could we delete it, and if it is, can we leave a comment about why it's necessary?